src/components/implementation/tests/micro_booter/micro_booter.c

```diff
@@ -56,6 +58,50 @@ static void
 tls_set(size_t off, unsigned long val)
 { __asm__ __volatile__("movl %0, %%gs:(%1)" : : "r" (val), "r" (off) : "memory"); }
 
+long long total_swt_cycles = 0LL;
```

Comment:
I kept the code for doing it in 2 ways. 1. Sum of cycles for all iterations. (commented, and in this case I needed that variable to be global.) 2. Total time in all iterations. I thought it would be useful. I'll remove it!